CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to PatternFly-ng
+
+We would love for you to contribute to PatternFly-ng and help make it even better than it is
+today!  Below is information on how to get started as a contributor to PatternFly-ng.
+
+ - [Code of Conduct](#coc)
+ - [Question or Problem?](#question)
+ - [Issues and Bugs](#issue)
+ - [Submission Guidelines](#submit)
+ - [Coding Rules](#rules)
+ - [Building the Code](#coding)
+ - [Releasing](#releasing)
+
+## <a name="coc"></a> Code of Conduct
+An open community is important to us, please help us maintain that by following our [Code of Conduct][coc].
+
+## <a name="question"></a> Do you have a question?
+There are a few things that can be done to resolve any questions or problems you might have.
+ - Search our [GitHub issues][github-issues]
+ - Join our patternfly-ng channel on [Slack](http://slack.patternfly.org)
+ - Join our mailing-list following the instructions on [patternfly.org](http://www.patternfly.org/community/)
+
+
+## <a name="issue"></a> Found a bug or issue?
+Please help us out if you find a bug by submitting an issue to our [GitHub Repository][github-issues].
+We also welcome [PR's][github-pr] if you'd like to submit a fix.
+
+## <a name="submit"></a> Submission guidelines
```

Comment:
I'm not sure whether this should be part of submission guidelines or another section about Submission review process, but I think we should explain that submissions will be reviewed against PatternFly standards.  The text I've written up about this:

Any Angular-ng patterns submitted for contribution will be reviewed by designers on this repo associated with the PatternFly project to ensure that contributions match the PatternFly design standards for each component.  This design review is used to make sure that it meets one of the following criteria:

- Matches the "gold standard" design specifications as designed by PatternFly
- Matches the design specifications provided for a certain project and has been made general enough to work for other projects
- Goes through a design review and subsequent matching design documentation issues are created for PatternFly to match this contribution